boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpuapp.dts

```diff
@@ -343,8 +343,8 @@ zephyr_udc0: &usbhs {
 
 /* Trim this RAM block for making room on all run-time common S2RAM cpu context. */
 &cpuapp_ram0 {
-	reg = <0x22000000 (DT_SIZE_K(32)-32)>;
```

Comment:
temp. block until input from @carlescufi - this places random burdens on VV

Reply:
No, we need it for the release that will be tomorrow.

Reply:
Keep in mind that it is a separate commit that adds 4 spaces and should be more than trivial to handle during the upmerge (either way - noup commits create conflicts).

Reply:
This places the blame of going through noup's on whomever is doing the upmerge, we have over 100 noup commits in sdk-zephyr, it is not nor should it be their responsibility to go through these 100+ commits, read their descriptions, apply possible action and then try to get them to apply. If things are needed then the required commits should be cherry-picked with them, if that takes 1 or 30 commits isn't VV's problem, it's on the person submitting to deal with and bring those in. An exception is being made for this due to urgency, future usage will be blocked and users will be asked to fromlist/fromtree cherry pick the required commits for a commit to be cleanly applied